log.go

```diff
@@ -19,7 +19,15 @@ func initLog() {
 		logrus.SetFormatter(newLogPrettyFormatter())
 	}
 
-	logrus.SetLevel(logrus.DebugLevel)
+	logLevel := "debug"
```

Comment:
Let's make it `info` by default. I made it `debug` because it wasn't configurable.